.github/workflows/feature_6_docs.yml

```
@@ -0,0 +1,63 @@
+name: Develop > Tiny docs 6.0
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - '*'
+    branches:
+      - 'feature/6**'
```

Should this be something like this instead?
```suggestion
      - 'feature/6/**'
```

---

Its to give people some room to have minor version if they want to. What do you think should we @lnewson ?

---

Ahh fair enough... I guess I was just a little concerned about things like `feature/6-DOC-1234` being matched, but it's probably not something to worry about.